tutorials/setup-elastic-metal-proxmox-cluster-with-private-networks/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+meta:
+  title: Setting up a Proxmox cluster on Elastic Metal servers using Private Networks
+  description: This page explains how to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+content:
+  h1: Setting up a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+  paragraph: This page explains how to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+categories:
+  - dedicated servers
+  - elastic-metal
+tags: Proxmox Elastic-Metal KVM-over-IP virtual-machine container
+dates:
+  validation: 2022-01-10
+  posted: 2022-01-10
+---
+
+## Proxmox - Overview
+
+[Proxmox Virtual Environment](https://www.proxmox.com/en/proxmox-ve) - or Proxmox VE - is a server virtualization platform, based on Debian Linux. It allows the deployment and management of [KVM-based](https://www.linux-kvm.org/page/Main_Page) virtual machines or [LXC](https://linuxcontainers.org/) containers.
+In this tutorial, we show you how to install Proxmox on your Elastic Metal servers, configure the Private Network via the Proxmox interface, and create a cluster of virtual machines across the two Elastic Metal servers which communicate via the Private Network. All the virtual machines can be managed from a single Proxmox interface, with the Private Network ensuring a high level of security and low latency.
+The tool provides a REST-API as well as a graphical web interface to manage the virtual machines. It is also licensed under the [GNU Affero General Public License, version 3](https://www.gnu.org/licenses/agpl-3.0.html).
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have at least two [Elastic Metal servers](https://www.scaleway.com/en/dedibox/)
+- You have created a [Private Network](/elastic-metal/how-to/use-private-networks/) and attached your Elastic Metal servers to it
```

Comment:
maybe add a link for "how to attach" the servers

Reply:
Good point. It seems we have no doc about it. shall we create one?